The-SMIRNOFF-force-field-format.md

```diff
@@ -1,103 +1,288 @@
+```
+Currently, only `potential="harmonic"` is supported, where we utilize the standard harmonic functional form:
+```
+U(r) = (k/2)*(theta-angle)^2
 ```
+Later revisions will add support for additional potential types and the ability to support arbitrary algebraic functional forms.
+If the `potential` attribute is omitted, it defaults to `harmonic`.
 
-**AMBER functional forms drop the factor of 2 in the angle energy term, which we elect not to do here.**
-Thus, comparing a SMIRNOFF file to a corresponding AMBER parameter file or .frcmod will make it appear that force constants here are twice as large as they ought to be.
+**Note that AMBER and CHARMM define a modified functional form**, such that `U(r) = k*(theta-angle)^2`, so that force constants would need to be multiplied by two in order to be used in the SMIRNOFF format.
 
-### PROPER TORSIONS
+### `<ProperTorsionForce>``
 
-Torsions are implemented as a [`PeriodicTorsionForce`](http://docs.openmm.org/7.0.0/userguide/theory.html#periodictorsionforce) tag with child tags for `Proper` (discussed here) and `Improper` (discussed below) parameters, for example:
+Proper torsions are specified via a `<ProperTorsionForce>...</ProperTorsionForce>` block, with individual `<Proper/>` tags containing attributes specifying the periodicity (`periodicityN`), phase (`phaseN`), barrier height (`kN`), and optional torsion multiplicity to divide the barrier height by (`idivfN`):
 ```XML
-<PeriodicTorsionForce phase_unit="degrees" k_unit="kilocalories_per_mole">
+<ProperTorsionForce potential="charmm" phase_unit="degrees" k_unit="kilocalories_per_mole">
    <Proper smirks="[a,A:1]-[#6X4:2]-[#6X4:3]-[a,A:4]" idivf1="9" periodicity1="3" phase1="0.0" k1="1.40"/>
-...
    <Proper smirks="[#6X4:1]-[#6X4:2]-[#8X2:3]-[#6X4:4]" idivf1="1" periodicity1="3" phase1="0.0" k1="0.383" idivf2="1" periodicity2="2" phase2="180.0" k2="0.1"/>
-...
-</PeriodicTorsionForce>
+   ...
+</ProperTorsionForce>
 ```
-
 Here, child `Proper` tags specify at least `k1`, `phase1`, and `periodicity1` attributes for the corresponding parameters of the first force term applied to this torsion.
-However, additional values are allowed in the form kN, phaseN, and periodicityN, where all N values must be consecutive (e.g. no `k1` and `k3` values without a `k2` value) but `N` can go as high as necessary.
+However, additional values are allowed in the form `kN`, `phaseN`, and `periodicityN`, where all `N` values must be consecutive (e.g., it is impermissible to specify `k1` and `k3` values without a `k2` value) but `N` can go as high as necessary.
+
+Currently, only `potential="charmm"` is supported, where we utilize the functional form:
+```
+U = \sum_{i=1}^N k_i * (1 + cos(periodicity_i * phi - phase_i))
+```
+**Note that AMBER defines a modified functional form**, such that `U = \sum_{i=1}^N (k_i/2) * (1 + cos(periodicity_i * phi - phase_i))`, so that barrier heights would need to be divided by two in order to be used in the SMIRNOFF format.
```

Comment:
The division mechanism you mentioned above, `idivf1`, `idivf2`, ..., should *not* be used here, to scale between AMBER and SMIRNOFF formats, right?

Reply:
If you are converting over AMBER parameters, you would want to both divide the barrier heights by two *and* copy over the AMBER `idivf` terms. Should we simplify this by adding a `potential="amber"` form as well, so only `idivf` is needed?

Reply:
I think so, yes, but not a big deal.